src/sentry/interfaces/security.py

```diff
@@ -112,9 +111,6 @@ def get_tags(self):
     def get_title(self):
```

Comment:
All the functions like `from_raw`, `get_tags`, `get_title`, ... are being used only by `event_manager.process_csp_report`, which is now dead. They can be completely removed. 

It seems that only `to_python` needs to remain (although it can be simplified in a _follow-up_) for `sentry.interfaces.base.get_interfaces` to work.